### PR TITLE
Validations added to create multiple leads creation scenario.

### PR DESCRIPTION
**Pull request by MelissaRoman06**

### CODE REVIEW

File: src/test/java/com/foundation/salesforce/steps/LeadsSteps.java [hunk 1]
```diff
@@ -223,4 +224,15 @@ public void aUserSpecifiesCompanyAndLastName(String company, String lastName) {
     public void aUserProvidesTheFollowingJson(String jsonBodyString) {
         requestManager.buildSpec(jsonBodyString);
     }
+
+    /**
+     *
```
Comment (alszla) [inline src/test/java/com/foundation/salesforce/steps/LeadsSteps.java:230]: Please add a brief description.

**Review by alszla — APPROVED** [review]

File: src/test/java/com/foundation/salesforce/steps/LeadsSteps.java (hunk 1 above)
Comment (MelissaRoman06) [reply src/test/java/com/foundation/salesforce/steps/LeadsSteps.java:230]: done

**Review by TheYisusGithub — APPROVED** [review]
That was ok.

File: src/test/java/com/foundation/salesforce/hooks/LeadHooks.java [hunk 2]
```diff
@@ -71,7 +71,7 @@ public void deleteLeadAfterCreation() {
     /**
      * Sets context's lead id after LeadCreation scenario.
      */
-    @After(value = "@LeadCreation, @SeveralLeadsCreation", order = 1)
+    @After(value = "@LeadCreation,  @MultipleLeadsCreation", order = 1)
```
Comment (aburgosu) [inline src/test/java/com/foundation/salesforce/hooks/LeadHooks.java]: delete white space

Comment (MelissaRoman06) [reply src/test/java/com/foundation/salesforce/hooks/LeadHooks.java]: done

**Pull request merged**